### ISSUE TITLE
Podman support

### ISSUE THREAD

**Issue by l0rd** [joyrex2001/kubedock#34]
### Problem Description

When trying to run podman on a host where I have started kubedock I get the following error

```bash
$ podman --remote --url "tcp://127.0.0.1:2475" run --name httpd -d -p 8080:8080 python python -m http.server 8080
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman socket: ping response was 404
```

And from kubedock logs

<img width="929" alt="image" src="https://user-images.githubusercontent.com/606959/232913542-336f9fa3-8560-4722-9354-9e47c8d514d7.png">

### How to reproduce it

The issue can be reproduced on OpenShift Developer Sandbox (a Red Hat Developer account is required but it's free) using the following link: https://workspaces.openshift.com/f?url=https://github.com/l0rd/micronaut-sample

### Video recording

https://user-images.githubusercontent.com/606959/232914999-8a171ca7-67ea-401c-83cf-a614d1f76c34.mp4

**Comment by joyrex2001** [comment]
Initial work has been started where podman routes can be implemented [here](https://github.com/joyrex2001/kubedock/tree/11fd549125aa1c5236a103576680b737a6aa2571/internal/server/routes/libpod), adjacent to the docker api implementation.

**Comment by l0rd** [comment]
> Initial work has been started where podman routes can be implemented [here](https://github.com/joyrex2001/kubedock/tree/11fd549125aa1c5236a103576680b737a6aa2571/internal/server/routes/libpod), adjacent to the docker api implementation.

Amazing. I have been testing a simple `podman run` and the pod got created successfully. I have also tested with `podman run -ti busybox` and `podman run -d -P httpd` but the first didn't start and the second didn't expose any port (which is perfectly fine for an initial support).

**Comment by agiertli** [comment]
Hi @joyrex2001 

I noticed that 'podman generate kube' didn't work with kubedock. Kubedock logs showed 404, which suggests the feature is simply not implemented.

Since this GH issue is about general podman support, would it be OK to add "podman generate kube" requirement as part of  this ticket?

**Comment by joyrex2001** [comment]
@agiertli yes, makes sense to track it in this ticket (sorry for the late reply).

I just released [0.13.0](https://github.com/joyrex2001/kubedock/releases/tag/0.13.0), which means for podman support:
* added support for `podman exec` interactive mode
* added support for `podman restart`
* fixed `podman cp` error
* fixed `podman ps` ports

**Comment by csfercoci** [comment]
what other commands are missing ?
